frontend/src/components/Table/MoneyCell.tsx

```diff
@@ -39,4 +50,41 @@ export const EditableMoneyCell = ({
   );
 };
 
-export const MoneyCell = ({ cell: { value } }: CellProps<IProperty, number>) => formatMoney(value);
+/**
+ * Provides a currency editing input if the user is allowed to edit the property and if the property is land/subdivision.
+ * @param param0 Configuration properties for this editable money cell.
+ */
+export const EditableLandMoneyCell = ({
+  namespace = 'properties',
+  suppressValidation,
+  cell,
+}: IEditableCellProps) => {
+  const context = useFormikContext();
+
+  const property = cell.row.original;
+  const canEdit =
+    useKeycloakWrapper().canUserEditProperty(property) &&
+    cell.row.values.propertyTypeId !== PropertyTypes.BUILDING;
+
+  return canEdit ? (
+    <FastCurrencyInput
+      formikProps={context}
+      suppressValidation={suppressValidation}
+      field={`${namespace}.${cell.row.id}.${cell.column.id}`}
+    ></FastCurrencyInput>
+  ) : (
+    <TooltipWrapper
+      toolTipId={`${namespace}.${cell.row.id}.${cell.column.id}`}
+      toolTip="You may only edit a property if it belongs to your agency and it is not part of an SPP Project."
```

Comment:
should this tooltip also mention that it can't be edited because it's a building on land?